test/cmd/apply.sh

```diff
@@ -27,75 +27,75 @@ run_kubectl_apply_tests() {
   kube::log::status "Testing kubectl apply"
   ## kubectl apply should create the resource that doesn't exist yet
   # Pre-Condition: no POD exists
-  kube::test::get_object_assert pods "{{range.items}}{{$id_field}}:{{end}}" ''
+  kube::test::get_object_assert pods "{{range.items}}{{$ID_FIELD}}:{{end}}" ''
```

Comment:
this commit is pretty huge at + 1,421 / - 1,421, I'm a little concerned that it would be easy to miss one of these.

Reply:
Sure, it is, as if we agree on this, I'd like to separate those uppercase replacements in another PR.